apps/super-app/src/services/SupportAboutServices.ts

```diff
@@ -0,0 +1,6 @@
+export interface SupportAbout {
```

Comment:
> apps/super-app/src/services/SupportAboutServices.ts

I don't quite understand the file name. Is this a (software) service that serves `SupportAbout`s? If so, then let's drop the `Services` suffix; it's redundant with the directory name. In either case, do stick with the singular noun for the file name.

Maybe this is a (real life service), but I'm not sure how to interpret the name.